src/main/java/org/agbrothers/englishbot/messagebuilder/DeletingWord.java

```diff
@@ -0,0 +1,36 @@
+package org.agbrothers.englishbot.messagebuilder;
+
+import org.agbrothers.englishbot.entity.Word;
+import org.agbrothers.englishbot.service.DictionaryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import static org.agbrothers.englishbot.constant.ButtonLabel.REMOVE_WORD;
+
+@Component
+public class DeletingWord extends MessageBuilder {
```

Comment:
You should be consistent in naming. Please, look at the other MessageBuilder classes and name this accordingly